src/scripts/libs/get-local-storage-language.ts

```diff
@@ -0,0 +1,18 @@
+import config from '../config/main';
+
+import {Language} from '../types/language';
+
+export default function getLocalStorageLanguage(): Language | null {
+  const storedLanguage = localStorage.getItem(config.localStorageLanguage);
+  const languages = Object.values(Language);
+
+  const availableLanguage =
+    storedLanguage &&
+    languages.find(
+      language =>
+        language.substr(0, 2).toLowerCase() ===
```

Comment:
We don't need the substr stuff here. Unlike in the getBrowserLanguage function the language in the localStorage should match the defined languages exactly.